Dockerfile.localdev

```diff
@@ -0,0 +1,98 @@
+# (Keep the version in sync with the node install below)
```

Comment:
Question: What's the value in having a separate dockerfile? The main one ought to work just fine for this.